docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+This point is called the **Track Point**.
+The trajectory is formed by connecting the track points collected across successive frames.
+
+!!! info inline end "Track Point examples"
+    === "0.1, 0.1"
+        ![Offset x=0.1, y=0.1](usage-ui/offset-x-0-1-y-0-1.png){ align=left }
+
+    === "0.8, 0.3"
+        ![Offset x=0.8, y=0.3](usage-ui/offset-x-0-8-y-0-3.png){ align=left }
+
+The position of the track point within the bounding box can be configured using the **Offset** attribute of sections.
+This offset is defined separately for the x- and y-axes, with values ranging from `0` to `1`.
+These values determine the relative position of the track point within the bounding box, providing flexibility in track point placement.
+
+Since the offset can be individually configured for each section, tracks intersecting different sections can be finely tuned.
+This allows precise customization of the trajectory data based on the requirements of each section.
+
+**Flows** define traffic flows to be analyzed.
+As explained in [Flows](#flows), a flow is defined by two sections: a starting section and an ending section.
+Tracks that intersect both sections of a flow can be assigned to that flow.
+To achieve this, the events belonging to a track are sorted chronologically based on their time of occurrence.
+The pair of events with the longest time interval between them is used to assign the track to the flow corresponding to the sections of both events.
+The assignment of a track to a flow is called **Track Assignment**.
+
+## User Interface
+
+The user interface is divided into four main areas: the configuration bar, workspace, visualization layers, and filter area.
+
+The **Configuration Bar** consists of four sections:
+
+1. **Project**: Manage project metadata and save or load `.otconfig` files.
+2. **Tracks/Video**: Add input files, where only track files can be added, while video files can be both added and removed. An overview of loaded video files is also displayed.
+3. **Sections/Flows**: Create, edit, and delete sections and flows. The flow names are shown with numbers indicating the number of assigned tracks per flow.
+4. **Analysis**: Start the export of analyzed data.
+
+The **Workspace** displays the tracks and video frames.
+It is also where the geometry of sections can be created or edited.
+Below the workspace, the **Track Statistics** section provides key statistics about the displayed tracks.
+
+Tracks can be customized using different **Visualization Layers**, while the number of displayed tracks can be adjusted through the **Visualization Filters** located below the workspace.
+
+![User Interface of OTAnalytics](usage-ui/user-interface.png)
+
+## Project Setup
+
+The project must first be named.
+The project name is entered in the corresponding Name field.
+The start time (date and time) of the first video must then be entered in the corresponding Start date fields.
```

Comment:
Why does the user have to enter the time at all? Pls explain...

When I enter hours, minutes and seconds below 10, they are displayed as single digits (looks weird)